### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by medsonmoombe**
## In this pull request the following were implemented:
 
-  For each URL I created:
    - A route.
    - An action in the correct controller.
    - A view file.

-  View files  includes only placeholders written in plain HTML, e.g:
    - Here is a list of posts for a given user

-  Added code to handle **only** the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'posts/index'
+  get 'posts/show'
+  get 'users/index'
+  get 'users/show'
```
Comment (TedLivist) [inline config/routes.rb]: - [ ] Do we really need these routes when we have defined them below? To keep your code clean, kindly remove these unnecessary routes.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @medsonmoombe,

Good job so far!
✔️ Good documentation
✔️ Creation controllers
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medsonmoombe** [comment]
## Implemented the changes required by the reviewer:
- Removed the unnecessary routes.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @medsonmoombe,

✔️ Simpler routing

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**